src/mempool/mempool.rs

```diff
@@ -149,6 +170,43 @@ impl Mempool {
         return true;
     }
 
+    async fn insert(&mut self, message: MempoolMessage) {
```

Comment:
Let's emit some stats here, number of inserts tagged by success/failure and message type, and a guage for the current size of the mempool by shard.

Reply:
https://github.com/farcasterxyz/snapchain/pull/200/commits/4842eafcd741eb9c951b24b86085989ad9280544

Reply:
Let me know if you want additional stats or tweaks to the formatting